DbSetup-kotlin/src/main/kotlin/com/ninja_squad/dbsetup_kotlin/DbSetupBuilder.kt

```diff
@@ -65,7 +65,7 @@ class DbSetupBuilder {
      * }
      * ```
      */
-    fun insertInto(table: String, configure: Insert.Builder.() -> Unit) {
+    inline fun insertInto(table: String, configure: Insert.Builder.() -> Unit) {
```

Comment:
It's probably a useless premature optimization. I'm not sure if I should leave it there or not. But the difference with the others is that this one takes a function as argument. And since Kotlin doesn't compile to Java 8 byte-code, but to Java 6, that means that every usage of insertInto will generate a nested class (so, more byte-code/.class files generated by the compiler), and will need a new instance at runtime for every call (and thus another GC).
Since the performance of DbSetup iitself s negligible compared to the time taken to actually execute the SQL statements, and since it's not used in constrained environments like android, it doesn't really matter.

To check I'm not saying bullshit, compile the tests without the inline keyword, and notice the presence of the class file `DbSetupBuilderTest$should insert$1$1.class`. Then recompile with the inline keyword, and notice its absence.

Inlining the other methods would just remove a method call, but that's useless, because the JIT will inline such code if needed.

Note: I didn't guess that by myself. It's explained in "Kotlin in Action", and also in this video: https://realm.io/news/oredev-jake-wharton-kotlin-advancing-android-dev/